scripts/install/install_bmi.sh

```diff
@@ -0,0 +1,58 @@
+set -x
+
+### START BMI setup
+
+TOP_DIR=$(cd $(dirname "$0") && pwd)
+
+# Move to main ims directory for install from script location
+pushd $TOP_DIR/../../
+
+### Install BMI
+
+user=`whoami`
+sudo mkdir -p /var/log/bmi/logs /etc/bmi /opt/bmi /var/lib/tftpboot
+sudo chown -R $user:$user /etc/bmi /var/log/bmi /opt/bmi /var/lib/tftpboot /etc/tgt
+
+### Copy config/template files
+cp bmi_config.cfg.test /etc/bmi/bmiconfig.cfg
+cp ims/ipxe.temp /etc/bmi/ipxe.cfg
+cp ims/mac.temp /etc/bmi/pxelinux.cfg
+
+virtualenv .bmi_venv
+source .bmi_venv/bin/activate
+pip install python-cephlibs
```

Comment:
Since we're running bmi in a virtual environment, we need to install python-cephlibs for rados client in bmi to work.

Also, ``apt-cache showpkg ceph-common | grep python-cephlibs`` doesn't yield anything